src/library_pthread.js

```diff
@@ -613,6 +613,13 @@ var LibraryPThread = {
     PThread.threadInit();
   },
 
+  __pthread_create_proxied__sig: 'iiiii',
+  __pthread_create_proxied__proxy: 'sync',
+  __pthread_create_proxied__deps: ['__pthread_create_js'],
```

Comment:
Can you mark this as `__pthread_create_proxied__internal: true,`?

Reply:
Any also maybe call it `$pthreadCreateProxied` so it clear its a JS-only function?

Reply:
OK, done with commit 495f49399d8837b999f91fcd2e63112a235eab72.